lightkurve/lightcurve.py

```diff
@@ -1070,6 +1072,64 @@ def estimate_cdpp(self, transit_duration=13, savgol_window=101,
         cdpp_ppm = np.std(mean) * 1e6
         return cdpp_ppm
 
+    def query_solar_system_objects(self, cadence_mask='outliers', radius=None, sigma=3, cache=True):
+        '''Query to find what solar system are in a lightcurve or targetpixel file.
```

Comment:
Please use triple double-quotes for docstring, and update/clarify the description (e.g. the reference to a target pixel file is old).